src/Resources/contao/library/Contao/Search.php

```diff
@@ -233,39 +233,13 @@ public static function indexPage($arrData)
 
 		unset($arrSet);
 
-		// Remove special characters
-		$strText = preg_replace(array('/- /', '/ -/', "/' /", "/ '/", '/\. /', '/\.$/', '/: /', '/:$/', '/, /', '/,$/', '/[^\w\'.:+-]/u'), ' ', $strText);
-
 		// Split words
-		$arrWords = preg_split('/ +/', Utf8::strtolower($strText));
+		$arrWords = self::splitIntoWords(Utf8::strtolower($strText), $arrData['language']);
```

Comment:
Should we make this `static::splitIntoWords()` and set the visibility of the method to `protected` so people can overwrite it if needed?

Reply:
I’m not sure if this is needed. What is the use case for extending the `Search` class?